justfile

```diff
@@ -79,6 +79,7 @@ configure-force-error: (configure "-DWARNINGS_AS_ERRORS=ON")
 # Use alternative compiler and enable LTO (test compiling only, won't run properly)
 configure-clang-lto: (configure "-DENABLE_LTO=ON -DCMAKE_CXX_COMPILER=clang++ -DCMAKE_C_COMPILER=clang") 
 
+configure-clang-lto-fail-on-warning: (configure "-DENABLE_LTO=ON -DCMAKE_CXX_COMPILER=clang++ -DCMAKE_C_COMPILER=clang -DWARNINGS_AS_ERRORS=ON")
```

Comment:
This could prob be done as
```
configure-clang-lto-fail-on-warning: (configure-clang-lto "-DWARNINGS_AS_ERRORS=ON")
```
Please test if that works localy

Reply:
nvm we can test it on the fly too

Reply:
Ok so that didnt work, I now went back to the longer-but-working version